LethalAPI.Core/Patches/GameNetworkManagerJoinPatch.cs

```diff
@@ -1,27 +1,29 @@
 ﻿// -----------------------------------------------------------------------
-// <copyright file="GameNetworkManager.cs" company="Lethal Company Modding Community">
-// Copyright (c) Lethal Company Modding Community. All rights reserved.
+// <copyright file="GameNetworkManagerJoinPatch.cs" company="LethalAPI Modding Community">
+// Copyright (c) LethalAPI Modding Community. All rights reserved.
 // Licensed under the GPL-3.0 license.
 // </copyright>
 // -----------------------------------------------------------------------
 
-namespace LCAPI.Core.Patches;
+// ReSharper disable InconsistentNaming
+namespace LethalAPI.Core.Patches;
 
 using System.Linq;
 
 using HarmonyLib;
 using Steamworks;
 using Steamworks.Data;
 
-[HarmonyPatch(typeof(GameNetworkManager))]
-internal static class GameNetworkManagerPatches
+/// <summary>
+/// Patches <see cref="GameNetworkManager.SteamMatchmaking_OnLobbyCreated"/> to add modded flags to the lobby.
```

Comment:
This should be changed to "Patches ... to add extra lobby metadata for using with the below patch (see cref to the patch below)"

Reply:
I don't think the `(see cref to the patch below)` is necessary, as it will show up highlighted anyways. I will add a seealso to help remedy this.